dev/ansible/03_run_tests.yml

```diff
@@ -0,0 +1,72 @@
+- name: Minio healthchecks pass
+  uri:
+    url: http://localhost:8500/v1/health/checks/minio?filter={{ '(Status=="passing")' | urlencode }}
+    method: GET
+    return_content: yes
+    status_code: 200
+    body_format: json
+  register: result_minio
+  retries: 15
+  delay: 15
+  until: result_minio.json | length == 2
+  tags: test
+
+# https://unix.stackexchange.com/a/148698
+- name: Start local proxy on port 9999 to minio service
+  shell: nohup consul connect proxy -service minio-local -upstream minio:9999 -log-level debug </dev/null >/dev/null 2>&1 &
+  register: test_output
+
+- name: Debug test_output
+  debug:
+    msg: "{{ test_output }}"
+
+- name: Register proxy PID_ID
+  shell: lsof -t -i:9999
+  register: PROXY_PID_ID # kill later with smth like `sudo kill -TERM 4635`
+
+- name: Minio curl example. Verify that proxy to minio works
+  uri:
+    url: http://127.0.0.1:9999
+    status_code: 403
+  register: curl_output
+  until: curl_output.status == 403
+  retries: 10
+  delay: 5
+
+- name: Debug curl_output
+  debug:
+    msg: "{{curl_output}}"
+
+- name: Add minio config
+  shell: mc config host add I-AM-THE-CONFIG http://127.0.0.1:9999 minio minio123
```

Comment:
I would suggest something like minio-container or similar. I-AM-THE-CONFIG seems out of place.